docs/user/templates.rst

```diff
@@ -94,6 +94,12 @@ once the device connects to OpenWISP it will download and apply the
 default templates without the need of manual intervention from the network
 operators.
 
+**Side note:** To be precise, a default template is automatically assigned not
+to any new device but to any new device config. So if you have any
+device which was registered before a specific template was made a default
+template, you can delete its config and create a new one, then the new config
+will automatically have all current default templates assigned.
```

Comment:
We shall keep in mind this could change: https://github.com/openwisp/openwisp-controller/issues/480.